charts/immich/templates/machine-learning.yaml

```diff
@@ -1,6 +1,6 @@
 {{- define "immich.machine-learning.hardcodedValues" -}}
 global:
-  nameOverride: machine-learning
+  nameOverride: {{ (index .Values "machine-learning").nameOverride | default "machine-learning" | quote }}
```

Comment:
I don't think it's necessary to template the field here. Just remove it from hardcodedValues and set it directly (`nameOverride: machine-learning`) in the values.yaml. That should be a simpler change while still working for your needs.

Reply:
How come I missed it! surely this way is more concise and robust.